.github/workflows/ci-testing.yml

```diff
@@ -106,6 +106,11 @@ jobs:
         pip list
       shell: bash
 
+    - name: Install serve test dependencies
+      if: matrix.topic == 'serve'
+      run: |
+        pip install '.[all]' --pre --upgrade
```

Comment:
if you install the package you shall also uninstall it at the end, otherwise, it will be part of the cache and not sure about eventual collisions...

Reply:
Done :smiley: